mkdocs/compat.py

```diff
@@ -0,0 +1,21 @@
+import sys
```

Comment:
I'm really not a fan of re-creating this file. We got rid of it originally because it became a unmaintainable mess. The few compatability bits we do need are in:

https://github.com/mkdocs/mkdocs/blob/master/mkdocs/utils/__init__.py#L32

To give further context, I moved us to six which was much nicer from a code point of view, but it led us to having tons of install problems for users - it seems some distros special case six and you can't rely on having the version you specify in setup.py. I never quite understood why, but it caused a number of issues and emails on varying platforms.

Then when we removed six, @waylan opted for managing the different import paths within each file. I honestly am not a huge fan of that, but it is less tangled up than the compat.py was originally.

I'd personally be tempted to vendor six, if we did really want a file like this, then we would at least have something sorta-standard and maintained/well tested but that wasn't particularly popular when I suggested it.